en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,462 @@
+
+1. In the terminal, navigate to the `choreo-reading-list-application-example/reading-list-front-end` directory in the clone of your GitHub repository.
+
+2. Issue the following commands in the given order:
+
+    `npm i`
+
+    `npm run dev`
+
+    The front-end application will start running at http://localhost:5173/.
+
+3. Access the front-end application via http://localhost:5173/.
+
+4. Click Login, and sign in with the credentials of a user that you have created in Asgardeo.
+
+5. To allow your application to access your Choreo account, select the **User Account** checkbox and click **Allow**.
+
+    The application opens as follows.
+
+    ![Front-end application](../assets/img/get-started/front-end-application.png){.cInlineImage-half}
+
+6. Add three new reading items with different statuses.
+
+    For example, the details can be as follows:
+
+     | **Title**                 | **Author**        | **Status** |
+     |---------------------------|-------------------|------------|
+     | `The Museum of Innocence` | `Orhan Pamuk`     | `reading`  |
+     | `The Remains of the Day`  | `Kazuo Ishiguro`  | `to_read`  |
+     | `David Copperfield`       | `Charles Dickens` | `read`     |
+
+    To add each record, follow the steps given below:
+
+    1. Click **+ Add New**.
+
+    2. Enter values for the **Name**, **Author**, and **Status** fields.
+
+    3. Click **Save**.
+
+    Three tabs will appear for each status. To delete a reading list item, you can click **Delete** for it.
+
+8. Log out of the front-end application.
+
+9. Log in to the front-end application again with the credentials of a different user that you created in Asgardeo.
+
+    You will not see the reading list items that you previously entered.
+
+## Step 3: Create and publish a new version of the REST API
+
+Currently, the updates to the readinglist are saved only as long as the deployment status of the `readingList` API remains **Active**. If you undeploy and redeploy the `readingList` API, you cannot retrieve the reading list with the updates you made after the last deployment. To address this, let's connect the REST API to a MySQL database. You can then release the improved REST API as a new version where data persistence is the new feature offered to application users.
+
+### Step 3.1: Create a new version of the REST API
+
+To create a new version of the `readingList` REST API with data persistence, connect it to a MySQL database on a cloud platform and redeploy it by following the steps given below:
+
+1. In the Choreo Console, open the `readingList` component (if it is not already open) and click the **Deploy** icon to open the **Deploy** tab.
+
+2. Expand the list of versions and click **+ Create New**.
+
+    ![Create new version](../assets/img/get-started/create-new-version.png){.cInlineImage-threeQuarters}
+
+3. Select **version-2** as the associated GitHub branch and enter `2.0.0` as the version name. Next click **Create**.
+
+4. Click **Config & Deploy**.
```

Comment:
Shall we say why we are now seeing config& deploy?

Reply:
Done

Reply:
Reversing this change again. Now they have endpoint the Sandbox endpoint for REST API, and we have to click **Config and Deploy** in the first instance too.